Cabal/Distribution/PackageDescription/Check.hs

```diff
@@ -627,7 +626,7 @@ checkSourceRepos pkg =
         ++ "field. It should specify the tag corresponding to this version "
         ++ "or release of the package."
 
-  , check (maybe False System.FilePath.isAbsolute (repoSubdir repo)) $
+  , check (maybe False isAbsolute (repoSubdir repo)) $
```

Comment:
This is not the only place we use `isAbsolute`/`isRelative`. Do we need to change every occurrence?